docs/items/items.md

```diff
@@ -30,4 +30,29 @@ Registering an Item
 
 Items must be [registered][registering] to function.
 
-[registering]: ../concepts/registries.md#registering-things
+Coloring an Item
+----------------
+
+Items' textures can be programmatically colored. Many Vanilla items utilize this functionality. For example, leather helmets, spawn eggs, potions and other.
+
+### Item Color Handlers
+
+An item color handler is required to color an item and is an instance of `IItemColor`, which adds the following method.
+
+```java
+int getColorFromItemstack(
+  ItemStack stack, 
+  int tintIndex)
+```
+
+This method returns a hex representation of a color in an integer.
+
+Tint indexes are specified for faces of an element in item's model JSON file. A face without a tint index won't be colored, and therefore will not have its color handler called. Layer indexes are used as tint indexes for items whose models inherit from the `builtin/generted` model.
+
+### Registering an Item Color Handler
+
+Item color handlers must be registered by calling `ItemColors#registerItemColorHandler(IItemColor, Item)` to function. Same can be done for multiple items. An instance of `ItemColors` can be obtained by calling `Minecraft#getItemColors()`. 
```

Comment:
Change the method reference to

"`ItemColors#registerItemColorHandler(IItemColor, Item...)` (or `ItemColors#registerItemColorHandler(IItemColor, Block...)` for `ItemBlock`s)